Echo.txt

```diff
@@ -0,0 +1 @@
+test 
```

Comment:
Practicing terminal commands? 😄 Remove any files you're not using before handing in